docs/interpreter/pig.md

```diff
@@ -113,4 +113,6 @@ foreach c generate group as category, COUNT($1) as count;
 
 Data is shared between `%pig` and `%pig.query`, so that you can do some common work in `%pig`, and do different kinds of query based on the data of `%pig`. 
 Besides, we recommend you to specify alias explicitly so that the visualization can display the column name correctly. Here, we name `COUNT($1)` as `count`, if you don't do this,
-then we will name it using position, here we will use `col_1` to represent `COUNT($1)` if you don't specify alias for it. There's one pig tutorial note in zeppelin for your reference.
+then we will name it using position, here we will use `col_1` to represent `COUNT($1)` if you don't specify alias for it. 
+
+There's one pig tutorial note in zeppelin for your reference. And we also has one wiki page for more details of running pig in zeppelin. https://cwiki.apache.org/confluence/display/ZEPPELIN/Running+Pig+in+Apache+Zeppelin
```

Comment:
I think `zeppelin` and `pig` need to be changed to `Zeppelin` and `Pig`. 
And "we also has" should be "we also have" :D